builtins/web/abort/abort-controller.cpp

```diff
@@ -35,7 +35,7 @@ bool AbortController::abort(JSContext *cx, unsigned argc, JS::Value *vp) {
 
   RootedValue reason(cx, args.get(0));
   RootedObject signal(cx, JS::GetReservedSlot(self, Slots::Signal).toObjectOrNull());
-  if (!signal) {
+  if (signal == nullptr) {
```

Comment:
I'm not necessarily convinced that changing all the `[!]foo` checks to explicit comparisons increases clarity, personally. As such I'd mildly prefer we disable whatever check requires them. But at this point doing so would require a lot of work undoing all the changes, so I'll not insist on it.

Reply:
So this is the check:

https://clang.llvm.org/extra/clang-tidy/checks/readability/implicit-bool-conversion.html

I'm happy to revert the changes it shouldn't be too much work.

Reply:
Sure! Makes sense. Let me try this out.